src/posix/platform/netif.cpp

```diff
@@ -340,31 +339,55 @@ static void LogDebg(const char *aFormat, ...)
 }
 
 #if defined(__APPLE__) || defined(__NetBSD__) || defined(__FreeBSD__)
-static const uint8_t allOnes[] = {0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF,
-                                  0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF};
+
+static const uint8_t kAllOnes[] = {
+    0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff,
+};
 
 #define BITS_PER_BYTE 8
 #define MAX_PREFIX_LENGTH (OT_IP6_ADDRESS_SIZE * BITS_PER_BYTE)
```

Comment:
nit: make them constants? Maybe in a future PR.

Reply:
Yes. Will leave for future PR. We can  address these in smaller PRs over time.
There are actually more cases similar to this (constants in `posix` platform modules where `#define` or `enum` is used and we can change to `constexpr`).